src/Plugin/WebformElement/NetsEasyPaymentElement.php

```diff
@@ -108,13 +116,13 @@ public function alterForm(array &$element, array &$form, FormStateInterface $for
        * inject placeholder for nets gateway containing amount to pay.
        */
       if ($amount_to_pay > 0) {
-        $form['content']['#markup'] = $element['#checkout_page_description'];
+        $form['content']['#markup'] = $element['#checkout_page_description'] ?? NULL;
```

Comment:
Don't add the `content` element if it will be empty:

```suggestion
        if (!empty($element['#checkout_page_description'])) {
          $form['content']['#markup'] = $element['#checkout_page_description'];
        }
```

Consider using [`'#type' => 'html_tag'`](https://www.drupal.org/node/2982025) to wrap the content in a `div` element to enable styling in the future.